dlpy/applications.py

```diff
@@ -20,17 +20,44 @@
 
 import os
 import warnings
+import six
 
 from .sequential import Sequential
 from .blocks import ResBlockBN, ResBlock_Caffe, DenseNetBlock, Bidirectional
 from .caffe_models import (model_vgg16, model_vgg19, model_resnet50,
                            model_resnet101, model_resnet152)
 from .keras_models import model_inceptionv3
-from .layers import (InputLayer, Conv2d, Pooling, Dense, BN, OutputLayer, Detection, Concat, Reshape, Recurrent,
-                     GlobalAveragePooling2D, GroupConv2d, Conv2DTranspose)
+from .layers import (Input, InputLayer, Conv2d, Pooling, Dense, BN, OutputLayer, Detection, Concat, Reshape, Recurrent,
```

Comment:
I am assuming we are not using them here.

Reply:
Yes, GroupConv2d, Conv2DTranspose will come in the next push related to UNet and MobileNet